src/config.js

```diff
@@ -1,3 +1,3 @@
-let DEBUG = false;
+let DEBUG = true;
```

Comment:
In future we could expose this on the window object to allow this to be toggled at runtime.

Reply:
ops, this should not get merged with debug set to true

Reply:
toggling it from window global would be lovely but it will require replacing webrequest listeners.